Packs/FortiSIEM/IncidentFields/incidentfield-FortiSIEM_Incident_First_Seen.json

```diff
@@ -0,0 +1,29 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "FortiSIEM"
+    ],
+    "caseInsensitive": true,
+    "cliName": "incidentfirstseen",
```

Comment:
Please add prefix to cliname.
should be `fortisiemincidentfirstseen `

Reply:
Done.